server-spi/src/main/java/org/keycloak/urls/HostnameProvider.java

```diff
@@ -34,7 +34,7 @@ public interface HostnameProvider extends Provider {
      * Returns the URL scheme. If not implemented will delegate to {@link #getScheme(UriInfo)}.
      *
      * @param originalUriInfo the original URI
-     * @param uype type of the request
+     * @param type type of the request
```

Comment:
@ya-makariy type is duplicated here. "..type type..."

Reply:
Also in the following methods
To remove duplicates, need to create another issue + pr?

https://github.com/keycloak/keycloak/blob/a6dd9dc0f1605ce0ac2b424df10e15a6eff6ff70/server-spi/src/main/java/org/keycloak/urls/HostnameProvider.java#L58  
https://github.com/keycloak/keycloak/blob/a6dd9dc0f1605ce0ac2b424df10e15a6eff6ff70/server-spi/src/main/java/org/keycloak/urls/HostnameProvider.java#L78  
https://github.com/keycloak/keycloak/blob/a6dd9dc0f1605ce0ac2b424df10e15a6eff6ff70/server-spi/src/main/java/org/keycloak/urls/HostnameProvider.java#L100

Reply:
@abstractj - just to double-check on your previous comment: the first "type" is the name of the parameter, the description of the type is "type of the request". 

When removing "type" from the description of the parameter, the description would read "of the request". Is this what you intended?

Reply:
@ahus1 you are correct, thank's for bringing this to my attention.

@ya-makariy my apologies about the confusion @ahus1 is correct on his statement. Answering your question, it is not necessary to create a issue to fix the typos you mentioned.